### ISSUE TITLE
[Bug][a11y] Page menu "untitled top" and "untitled bottom"

### ISSUE THREAD

**Issue by sv-ohorvath** [mozilla-mobile/fenix#4217]
## Steps to reproduce
1. Enable TalkBack.
2. Open a tab, open the main menu.
3. When the main menu is displayed, observe how it is read.
### Expected behavior
The tab main menu is announced (e.g. pop-up window), with a number of listed items.
### Actual behavior
 "untitled top" and "untitled bottom" is announced the first time you open the tab main menu.
### Device information

* Android device: Nexus 5 (android 6.0.1)
*Not an issue on Pixel 3 (android 9),, Nokia 6 (Android 7.1.1)
* Fenix version: RC 1.1.

**Comment by mcarare** [comment]
@sv-ohorvath can you please check if this is still an issue?

**Comment by sv-ohorvath** [comment]
@mcarare Yes it's still reproducing but please note that it doesn't happen on devices with Android 7 or higher.
devices I've reproduced this with: HTC Desire 820 (Android 6.0.1), LG Nexus 5 (Android 6.0.1)

**Comment by bowiz2** [comment]
I'd like to give this a first issue shot if it's available!

**Comment by mcarare** [comment]
@bowiz2 Sure, go ahead! 😄

**Comment by bowiz2** [comment]
@sv-ohorvath I've not been able to reproduce this on an **emulated** Nexus 5 running Android 6.0.1. Tried using Talkback (Android Accessibility Suite) versions 8.1 (latest), 4.2 and 4.3.1 (both versions released around Android 6.0.1).

When I open the tab main menu, Talkback doesn't read anything out to me, unless I swipe right, in which it reads the first object of the list, "Close all tabs".
If I enable continuous reading while on the New Tab page, Talkback reads out whatever is shown on screen. When I open the tab main menu and enable continuous reading, the three options on the list read out correctly.

Could you check to see if the issue is still occurring?

**Comment by sv-ohorvath** [comment]
@bowiz2 I can still reproduce, but now it says "Firefox nightly on top", "Firefox nightly on bottom".

**Comment by corneliuze** [comment]
Hi @sv-ohorvath, @bowiz2 , @mcarare  I would appreciate if you can show me the flow by uploading a GIF, I have downloaded an android device with version 6.0 and I want to look into the issue but I can't seem to get what you're talking about.
Thanks

**Comment by sv-ohorvath** [comment]
@corneliuze  Open a website. In the navigation toolbar, tap the 3 dot (hamburger) button.
![ss](https://user-images.githubusercontent.com/4037709/84741923-6dd8f600-afb8-11ea-96c3-80a92b94b18b.jpg)

**Comment by corneliuze** [comment]
Hi @sv-ohorvath, I couldn't replicate the issue on Nexus 4, Android 6.0

**Comment by sv-ohorvath** [comment]
@ebalazs-sv You have a Nexus 5 device, could you please check if this issue is still reproducing?

**Comment by corneliuze** [comment]
I'll download the emulator and check.

**Comment by ebalazs-sv** [comment]
This is still reproducible on the latest Fenix Nightly Build #21740609 from 6/22 with Nexus 5 (android 6.0.1).
"Firefox nightly on top, Firefox nightly on bottom, Add-ons, Double-tap to activate" is announced the first time you open the tab main menu.

**Comment by mcarare** [comment]
Apparently on those Android/Talkback versions screen reader interprets opening the menu as a split screen and names the 2 apps that are on the split screen. In this case, both Fenix.
